src/engine/bank1.asm

```diff
@@ -787,7 +787,63 @@ LoadPlayerDeck: ; 6793 (1:6793)
 	ret
 ; 0x67b2
 
-INCBIN "baserom.gbc",$67b2,$7107 - $67b2
+INCBIN "baserom.gbc",$67b2,$6d84 - $67b2
+
+ConvertItemToPokemon:
+	ld c, a
+	ld a, [hl]
+	cp TRAINER_CARD
+	ret nz
+	push hl
+	ldh a, [hWhoseTurn]
+	ld h, a
+	ld l, c
+	ld a, [hl]
+	and $10
+	pop hl
+	ret z
+	ld a, e
+	cp $cc
+	jr nz, .checkForClefairyDoll
+	ld a, d
+	cp $00
+	jr z, .startRamDataOverwrite
+	ret
+.checkForClefairyDoll
+	cp $cb
+	ret nz
+	ld a, d
+	cp $00
+	ret nz
+.startRamDataOverwrite
+	push de
+	ld [hl], $06
+	ld bc, wCardBuffer1HP - wCardBuffer1
+	add hl, bc
+	ld de, $6db9
+	ld c, wCardBuffer1Unknown2 - wCardBuffer1HP
+.loop
+	ld a, [de]
+	inc de
+	ld [hli], a
+	dec c
+	jr nz, .loop
+	pop de
+	ret
+
+.dataToOverwrite
+	db $0a            ; hp
+	ds $07 
+	dw $0030          ; move1 name
+	dw $0041          ; move1 description
+	ds $03
+	db $04            ; move1 category
+	dw $4e35          ; move1 effect commands
+	ds $18
+	db UNABLE_RETREAT ; retreat cost
+	ds $0d
+
```

Comment:
.dataToOverwrite
	db 10            ; hp
	ds wCardBuffer1Move1Name - (wCardBuffer1HP + 1) 
	tx Text0030          ; move1 name
	tx Text0041          ; move1 description
	ds wCardBuffer1Move1Category - (wCardBuffer1Move1Description + 2)
 	db POKEMON_POWER            ; move1 category
 	dw $4e35          ; move1 effect commands
 	ds wCardBuffer1RetreatCost - (wCardBuffer1Move1EffectCommands + 2)
 	db UNABLE_RETREAT ; retreat cost
 	ds CARD_DATA_LENGTH - (wCardBuffer1RetreatCost + 1 - wCardBuffer1)
```

I put this up blindly, I wouldn't be surprised if there was an error somewhere. It's better to have these offsets labeled imo, so that adding a new field is automatic.

$4e35 is actually https://github.com/pret/poketcg/blob/master/src/data/effect_commands.asm#L1348